modules/related-posts/jetpack-related-posts.php

```diff
@@ -118,59 +167,50 @@ public function action_admin_init() {
 	 * @returns null
 	 */
 	public function action_frontend_init() {
-		// Add a shortcode handler that outputs nothing, this gets overridden later if we can display related content
+		// Add a shortcode handler that outputs nothing, this gets overridden later if we can display related content.
 		add_shortcode( self::SHORTCODE, array( $this, 'get_target_html_unsupported' ) );
-
-		if ( ! $this->_enabled_for_request() )
+		if ( ! $this->_enabled_for_request() ) {
 			return;
-
+		}
 		if ( isset( $_GET['relatedposts'] ) ) {
 			$excludes = array();
 			if ( isset( $_GET['relatedposts_exclude'] ) ) {
 				$excludes = explode( ',', $_GET['relatedposts_exclude'] );
 			}
-
 			$this->_action_frontend_init_ajax( $excludes );
 		} else {
 			if ( isset( $_GET['relatedposts_hit'], $_GET['relatedposts_origin'], $_GET['relatedposts_position'] ) ) {
 				$this->_log_click( $_GET['relatedposts_origin'], get_the_ID(), $_GET['relatedposts_position'] );
 				$this->_previous_post_id = (int) $_GET['relatedposts_origin'];
 			}
-
 			$this->_action_frontend_init_page();
 		}
-
 	}
-
 	/**
 	 * Adds a target to the post content to load related posts into if a shortcode for it did not already exist.
 	 *
 	 * @filter the_content
-	 * @param string $content
+	 * @param string $content Content.
```

Comment:
This is not a great parameter description. The variable name already says it's content. What kind of content? Please update it here and in the others, as well as Input, Post ID, and others.